tools/functions.sh

```diff
@@ -9,6 +9,9 @@
 # shellcheck disable=SC2034
 installdir="${rootdir:?}/build/install"
 
+# Docker registry which is used in all docker-related scripts
+DOCKER_REGISTRY=registry.gitlab.com/prpl-foundation/prplmesh/
```

Comment:
`functions.sh` is a kinda generic script, not sure it's the right place for this global.
Maybe we should have a spearate, sourcable, script for docker specific scripts

Reply:
I don't know... The only non-docker scripts that use functions.sh are:
- cppcheck-new-issues.sh
- upload_to_owncloud.sh - but this will be converted to Python
- run_easymesh_cert.sh - wasn't it going to be converted to Python as well?
- download_ipk.sh

For these, having some variable defined that is not used in that script shouldn't be an issue, should it? The script also defines some other things that are maybe not defined anywhere else.